app/mutations/application_review_step/deleteReviewCommentMutation.ts

```diff
@@ -12,16 +12,16 @@ import {
 import {ConnectionHandler} from 'relay-runtime';
 
 const mutation = graphql`
-  mutation deleteReviewCommentMutation(
-    $input: UpdateReviewCommentInput!
-    $applicationId: ID!
-    $version: String!
-  ) {
+  mutation deleteReviewCommentMutation($input: UpdateReviewCommentInput!) {
```

Comment:
```suggestion
  mutation deleteReviewCommentMutation($input: DeleteReviewCommentInput!) {
```

Reply:
Just realized there's also no test for this mutation

Reply:
About the suggested snippet - if I do that, I get
```
ERROR:
Variable '$input' was defined as type 'DeleteReviewCommentInput!' but used in a location expecting the type 'UpdateReviewCommentInput!'
```

Because ultimately it's an updateReviewComment mutation

Reply:
Right, the line below, nevermind then